container/image_test.py

```diff
@@ -350,6 +350,11 @@ def test_flattened(self):
       self.assertTarballContains(tar, [
         '.', '/usr', '/usr/bin', '/usr/bin/java', './foo'])
 
+  def test_flattened(self):
```

Comment:
add short doc for this def, its otherwise not clear what differs from the one above

Reply:
Done